src/libraries/Microsoft.PowerFx.Core/Parser/NamedFormula.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Parser
+{
+    internal class NamedFormula 
```

Comment:
How (and where) was this concept being represented before? Or is the first time we had a class representing named formulas?

Reply:
We were using a IEnumerable<KVP<IdentToken, Formula>>